tests/test_interface_ospf.rb

```diff
@@ -21,6 +21,8 @@
 
 # TestInterfaceOspf - Minitest for InterfaceOspf node utility class.
 class TestInterfaceOspf < CiscoTestCase
+  @feature = 'interface_ospf'
```

Comment:
I like your fix overall, my only nit is that you're overloading the term `feature`. I don't have a better suggestion though. `yaml` sort of indicates where/what this is for but that's not all that helpful either. If you can think of a better term then great but don't consider this comment a blocker.